tests/e2e/logs_test.go

```diff
@@ -0,0 +1,47 @@
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("E2E - Getting logs node", Label("logs"), func() {
+	It("Get the upstream cluster logs", func() {
+		By("Downloading tools to generate logs", func() {
+			_ = os.Mkdir("logs", 0755)
+			_ = os.Chdir("logs")
+			err := exec.Command("curl", "-L", "https://github.com/rancher/elemental-operator/releases/download/v1.0.0/elemental-support_1.0.0_linux_amd64", "-o", "elemental-support").Run()
```

Comment:
Maybe better to add a loop here, as the download/configuration code is pretty much the same for both `elemental-support` and the script.

And IMHO also better to use a variable here that can be configured from the workflow (inputs also?) as it too static here.

Reply:
Done!